libunwind/src/libunwind.cpp

```diff
@@ -12,6 +12,7 @@
 #include <libunwind.h>
 
 #include "config.h"
+#ifndef __wasm__
```

Comment:
Similar question as above here. If this file is basically empty on wasm, we should instead avoid adding it in the CMakeLists.txt.

Reply:
the problem is that the debugging code needs in libunwind

Reply:
i have included apis with minimal code.
https://github.com/llvm/llvm-project/blob/b5fee4962c347016a732a1310275651a8c74f23a/libunwind/src/libunwind.cpp#L442
It needs logAPIs in the file